watson-developer-cloud/discovery-components
# fix: set search facet result count as disabled prop

#### What do these changes do/fix?
The Search Facet matching results count is now disabled by the prop flag `showMatchingResults`. After speaking with @ColbyJohnIBM, we decided to hide the work from PRs #114 and #122 until a revealed API issue is resolved. Issue has been opened here: https://github.ibm.com/Watson-Discovery/wire-plugin-tracker/issues/591

#### How do you test/verify these changes?
- In Tooling: Link this to Tooling. See that the documents count is no longer shown.
- In Storybook: Turn off the knob for `Show matching results` and see the count is no longer shown.

**Before:**
<img width="190" alt="Screen Shot 2020-04-09 at 10 43 44 AM" src="https://user-images.githubusercontent.com/59846843/79154465-3457f480-7d95-11ea-8cf0-450a59fd703f.png">

**After:**
<img width="151" alt="Screen Shot 2020-04-13 at 2 42 19 PM" src="https://user-images.githubusercontent.com/59846843/79154351-feb30b80-7d94-11ea-8dc1-cdb674fd2520.png">

#### Have you documented your changes (if necessary)?
Tests have also been updated.

#### Are there any breaking changes included in this pull request?
